### ISSUE TITLE
Add ability to set region configuration

### ISSUE THREAD

**Issue by evanaze** [jkausti/target-s3#1]
When trying to use this plugin on my project I'm getting the error:
```
PermanentRedirect Message: The bucket you are attempting to access must be addressed using the specified endpoint. Please send all future requests to this endpoint.
```
According to [this](https://stackoverflow.com/questions/25027462/aws-s3-the-bucket-you-are-attempting-to-access-must-be-addressed-using-the-spec) StackOverflow post, this error pops up when you have the incorrect region set. This should be a configurable parameter for the plugin, and can simply be set in `pyarrow.fs.S3FileSystem` as described [here](https://arrow.apache.org/docs/python/generated/pyarrow.fs.S3FileSystem.html#pyarrow.fs.S3FileSystem).

**Comment by jkausti** [comment]
Thanks for submitting an issue! This should be quite straightforward to fix, I'll try to get it done today.

**Comment by jkausti** [comment]
@evanaze I have now added an optional aws_region setting so hopefully the target works for you now. Let me know if you encounter other issues.

These S3FileSystem -settings should probably be some kind of kwarg-setting instead of explicitly specified in the target, there are a lot of possible settings missing still. However I will need to dig a bit deeper into the Meltano SDK to understand how to add that kind of a setting. I'll create a new issue for that.

Closed by #2

**Comment by evanaze** [comment]
Thanks! I appreciate the quick fix on this.
